translations/english/core-commands.ts

```diff
@@ -88,14 +88,13 @@ export const translations: Translations = {
 		"User ${name} must be in the battle room already.": "",
 		"This room already has a player in slot ${target}.": "",
 		"${targetUser.name} is already a player in this battle.": "",
-		"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
 		"Player 2": "",
 		"Players could not be restored (maybe this battle already has two players?).": "",
 		"This game doesn't support /joingame": "",
 		"This game doesn't support /leavegame": "",
 		"You can only do this in unrated non-tour battles.": "",
 		"User ${targetUsername} not found.": "",
-		"${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+		"${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",
```

Comment:
It doesn't matter what the variables are called in translation. `displayReason` is clearer, though, so I'm fine with it.

Reply:
Oh, Thank you!
(I'm just modifying it to reflect the changes)

Reply:
I'm just modifying it to reflect the changes :)

Reply:
It... doesn't need to reflect the changes, but this is fine either way.